ftest.sh

```diff
@@ -147,8 +148,8 @@ $NFS_SERVER:$PWD $DAOS_BASE nfs defaults 0 0 # added by ftest.sh
 wq
 EOF
 sudo mount $DAOS_BASE
-rm -rf /tmp/Functional_$TEST_TAG/
-mkdir -p /tmp/Functional_$TEST_TAG/
+rm -rf $TEST_DIR/
```

Comment:
This will raise a [shellcheck](https://www.shellcheck.net/) [SC2115: Use "${var:?}" to ensure this never expands to /](https://github.com/koalaman/shellcheck/wiki/SC2115) error at some point in the future when our shellcheck version catches up to the latest.

This one seems like a good one to mitigate given that it's an `rm -rf` of a dir in a variable which if the variable is empty somehow will result in a removing of the entire file system (assuming the `rm` doesn't have a built-in safety for that -- some do).

Reply:
Thanks for the great feedback.  Fix applied.